source/release-notes.txt

```diff
@@ -10,6 +10,28 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.46.0
+----------------
+
+*Released April 4, 2025*
+
+New Features:
+
+- Enable export schema feature (:issue:`COMPASS-8819`)
+- Add in progress saving state (:issue:`COMPASS-9077`)
+- Show validation error details (:issue:`COMPASS-8864`, :issue:`COMPASS-8867`, :issue:`COMPASS-8865`)
+- Add validation action option for 'errorAndLog' with 8.1 (:issue:`COMPASS-8983`)
+
+Bug Fixes:
+
+- Allow deleting updatemany queries (:issue:`COMPASS-9121`)
+- Remove readPreferenceTags for database&collection lookups if the cluster is sharded and readPreferenceTags was set (:issue:`COMPASS-9111`)
+- Catch parse error on multi insert
```

Comment:
[question for tech review]

Do these have accompanying `COMPASS` issue numbers? Thanks!

Reply:
No jira issues for those 👍